gap/json.gi

```diff
@@ -50,47 +50,53 @@ InstallMethod(_GapToJsonStreamInternal, [IsOutputStream, IsString],
 function(o, s)
   if IsEmpty(s) then
     if IsStringRep(s) then
-      PrintTo(o, "\"\"");
+      WriteAll(o, "\"\"");
```

Comment:
According to the Codecov comments, this part of the code is not covered by the test suite.